cmd/openapi-gen/builder/openapi_test.go

```diff
@@ -45,13 +46,13 @@ var def = definition.Definition{
 	},
 	Results: []definition.Result{
 		{
-			Type: definition.Meta,
-			Headers: map[string]string{
-				"xxx": "X-Xxx",
-			},
+			Destination: definition.Meta,
+			// Headers: map[string]string{
+			//     "xxx": "X-Xxx",
+			// },
 		},
-		{Type: definition.Data},
-		{Type: definition.Error},
+		{Destination: definition.Data},
```

Comment:
why change

Reply:
Because the `Type` is not clear. The field is corresponding to `Source`. So `Destination` is better than `Type`.